pages/about.vue

```diff
@@ -1,39 +1,55 @@
 <template>
-  <div id="about">
-    <div id="info">
-      <div align="center">
-        <h1>{{ $t("about.welcome") }}</h1>
-      </div>
-      <div>
-        <h2>{{ $t("about.whatIsThisQ") }}</h2>
-        <p v-html="$t('about.whatIsThisA')"></p>
-      </div>
-      <div>
-        <h2>{{ $t("about.trustQ") }}</h2>
-        <p v-html="$t('about.trustA')"></p>
-      </div>
-      <div>
-        <h2>{{ $t("about.supportQ") }}</h2>
-        <p v-html="$t('about.supportA')"></p>
-      </div>
-      <div>
-        <contributors-list />
-      </div>
-      <twitter-testimonials />
-    </div>
-    <div id="widgets">
-      <div id="donate">
-        <iframe
-          src="https://ko-fi.com/theyokohamalife/?hidefeed=true&widget=true&embed=true&preview=true"
-          style="border: none; width: 100%; padding: 4px; background: #f9f9f9"
-          height="650"
-          title="theyokohamalife"
-        ></iframe>
-      </div>
-    </div>
+  <div
+    id="about"
+    class="about-page"
+    :class="$vuetify.breakpoint.mobile ? 'mobile' : ''"
+  >
+    <section class="about-header">
+      <AboutHeader />
+    </section>
+    <section class="timeline">
+      <AboutSiteTimeline />
+    </section>
+    <section class="midpage-widgets">
+      <v-row class="no-gutters">
+        <v-col cols="12" xs="12" md="6">
+          <AboutKofiWidget />
+        </v-col>
+        <v-col cols="12" xs="12" md="6">
+          <AboutTwitterTestimonials :tweets="tweets" turquoise />
+        </v-col>
+      </v-row>
+    </section>
+    <section class="trust">
+      <AboutTrust />
+    </section>
+    <section class="team">
+      <AboutTeam />
+    </section>
+    <section class="bottom-twitter-widget">
+      <AboutTwitterTestimonials :tweets="tweets" />
+    </section>
   </div>
 </template>
+<script>
+import TwitterAPI from "../services/twitter";
 
+export default {
+  data() {
+    return {
+      tweets: [],
+    };
+  },
+  async asyncData({ $axios }) {
```

Comment:
would `getTweets` be a better name?

Reply:
yeah I debated over that. `getTweets` is fine, can change it from `get` to that.

Reply:
I actuall meant instead of `asyncData` 😅

Reply:
Ah, asyncData is actually a [nuxt hook](https://nuxtjs.org/docs/2.x/features/data-fetching) that fetches data for the entire page. It's a 'magic name'.

Reply:
We could theoretically refactor this to have a different getTweets function that gets referenced here, but with the logic so straightforward it doesn't make sense yet I think.

Reply:
ahhhh fair enough. another magic name 🔥